Author: ptrcnull <git@ptrcnull.me>
Date:   Sun Feb 6 08:51:27 2022 +0100

    Remove "static" from strlcpy
    
    powerstat.c:339:15: error: static declaration of 'strlcpy' follows non-static declaration

diff --git a/powerstat.c b/powerstat.c
index cf5fc6e..7d03f66 100644
--- a/powerstat.c
+++ b/powerstat.c
@@ -336,7 +336,7 @@ static const int signals[] = {
  *  strlcpy()
  *	BSD strlcpy
  */
-static size_t strlcpy(char *dst, const char *src, size_t len)
+size_t strlcpy(char *dst, const char *src, size_t len)
 {
 	char *d = dst;
 	const char *s = src;
